src/sentry/conf/server.py

```diff
@@ -32,20 +41,24 @@ def gettext_noop(s: str) -> str:
 socket.setdefaulttimeout(5)
 
 
+T = TypeVar("T")
```

Comment:
minor nit: using `_T` here prevents autocomplete from trying to import these across modules.

Reply:
I can update that.